dask/array/slicing.py

```diff
@@ -786,8 +786,14 @@ def normalize_index(idx, shape):
     >>> normalize_index((Ellipsis, None), (10,))
     (slice(None, None, None), None)
     """
+    from .core import from_array
+
     if not isinstance(idx, tuple):
         idx = (idx,)
+    if len(idx) > 0 and len(shape) > 1:
```

Comment:
Can you add a one-line comment above this saying what this is for?

Reply:
Sure! I updated the docstring as well.